e2e/isolated/isolated_test.go

```diff
@@ -0,0 +1,247 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package isolated_test
+
+import (
+	"errors"
+	"fmt"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	osExec "os/exec"
```

Comment:
nit: https://rakyll.org/style-packages/#renames-should-follow-the-same-rules
```suggestion
	osexec "os/exec"
```